src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -118,6 +118,7 @@ public static class ScriptConstants
         public const string FeatureFlagDisableDiagnosticEventLogging = "DisableDiagnosticEventLogging";
         public const string FeatureFlagDisableMergedWebHostScriptHostConfiguration = "DisableMergedConfiguration";
         public const string FeatureFlagEnableWorkerIndexing = "EnableWorkerIndexing";
```

Comment:
Would this constant still be needed?

Reply:
Right - we should remove if no longer being used